lib/model/query/projects.js

```diff
@@ -112,9 +112,24 @@ const getAllByAuth = (auth, options = QueryOptions.none) => ({ all }) =>
 const getById = (id, options = QueryOptions.none) => ({ maybeOne }) =>
   _get(maybeOne, options.withCondition({ id }));
 
+const getAllByAuthWithForms = (auth, queryOptions = QueryOptions.none) => ({ Projects, Forms }) =>
+  Promise.all([
+    Projects.getAllByAuth(auth, queryOptions),
+    Forms.getAllByAuth(auth, QueryOptions.extended)
+  ])
+    .then(([ projects, forms ]) => {
+      const projectsWithForms = projects.reduce((acc, proj) => ({ ...acc, [proj.id]: { ...proj.forApi(), forms: [] } }), {});
+      for (const form of forms) {
+        if (form.projectId in projectsWithForms)
+          projectsWithForms[form.projectId].forms.push(form.forApi());
```

Comment:
`forms` used to contain a count of forms in the extended Project so maybe this should be called something else so that still works.